react/Wrapper.tsx

```diff
@@ -78,15 +79,17 @@ const Wrapper = withToast(function Wrapper(props: Props) {
     addToCartFeedback = 'toast',
     onClickBehavior = 'add-to-cart',
   } = props
-  const productContext: ProductContextState = useProduct()
+  const productContext = useProduct() ?? {}
   const isEmptyContext = Object.keys(productContext).length === 0
```

Comment:
```suggestion
  const productContext = useProduct()
  const isEmptyContext = Object.keys(productContext ?? {}).length === 0
```

I think it makes more sense doing like this, since every call to productContext bellow already checks if it's undefined.